Drivers/Src/BatteryBalancing.c

```diff
@@ -97,12 +106,19 @@ static void Balancing_GetICNumber(uint8_t i, uint8_t* ICNumber, uint8_t* ModuleN
  * @return  None
  */
 static void Balancing_SetDischargeBit(uint8_t module, cell_asic ic[]) { 
+	OS_ERR err;
 	uint8_t ICNumber = 0; 
 	uint8_t ModuleNumber = 0;
 	Balancing_GetICNumber(module, &ICNumber, &ModuleNumber);//Get IC and ModuleInIC number
-	LTC681x_rdcfg(NUM_MINIONS, ic);
 	
+	OSMutexPend(&MinionsASIC_Mutex, 0, OS_OPT_PEND_BLOCKING, NULL, &err);
+	assertOSError(err);
+	
+	LTC681x_rdcfg(NUM_MINIONS, ic);
```

Comment:
This function returns an error. Should we be returning a value if there is an error?

Reply:
Opened issue #280 for this